sources/packages/backend/workflow/test/utils/assessment-consolidated-data-utils.ts

```diff
@@ -0,0 +1,109 @@
+import { OfferingIntensity } from "@sims/sims-db";
+import {
+  AssessmentConsolidatedData,
+  OfferingDeliveryOptions,
+  Provinces,
+  YesNoOptions,
+} from "../models/assessment.model";
+
+export function getFakeAssessmentConsolidatedData(
+  offeringIntensity: OfferingIntensity,
```

Comment:
Is it worth providing these values as parameters? Are they not supposed to be defined at test level?